modules/products/bamboo/helm.tf

```diff
@@ -94,3 +94,9 @@ resource "helm_release" "bamboo_agent" {
     }),
   ]
 }
+
+resource "random_string" "random" {
+  length  = 10
```

Comment:
Is ten a bit excessive? Maybe 5 is enough?

Reply:
Yes, but still this is part of a path and is not visible at all (even for us) - the default is 16 and I made it 10.